src/main/java/backend/json/ReadTask.java

```diff
@@ -27,21 +30,40 @@ public ReadTask(String repoId, CompletableFuture<Model> response) {
 
     @Override
     public void run() {
-        Model model = load(repoId);
-        response.complete(model);
+        try {
+            Model model = load(repoId);
+            response.complete(model);
+        } catch (RepoStoreException e) {
+            logger.error(HTLog.format(repoId, "Failed to load from store"));
```

Comment:
Try not to use `failed` in error messages as it makes failed tests harder to find in the ci build logs.